devbox.go

```diff
@@ -49,13 +49,26 @@ func Open(dir string) (*Devbox, error) {
 		return nil, errors.WithStack(err)
 	}
 
+	// if dir is current directory, then get the full path
+	if dir == "." {
+		var err error
+		dir, err = os.Getwd()
+		if err != nil {
+			return nil, errors.WithStack(err)
+		}
+	}
+
 	box := &Devbox{
 		cfg:    cfg,
 		srcDir: dir,
 	}
 	return box, nil
 }
 
+func (d *Devbox) SourceDir() string {
```

Comment:
I'm not sure this needs to be public – I think you have to make it public right now because you implemented a lot of the logic outside of the Devbox SDK – if you move that logic to be internal, then you don't need to expose this.